modules/intensity_transform/include/opencv2/intensity_transform.hpp

```diff
@@ -71,8 +72,20 @@ CV_EXPORTS_W void autoscaling(const Mat input, Mat& output);
 */
 CV_EXPORTS_W void contrastStretching(const Mat input, Mat& output, const int r1, const int s1, const int r2, const int s2);
 
+/**
+ * @brief Given an input color image, enhance low-light images using the BIMEF method (@cite ying2017bio @cite ying2017new).
+ *
+ * @param input input color image.
+ * @param output resulting image.
+ * @param mu enhancement ratio.
+ * @param k exposure ratio or NULL if it must be estimated.
+ * @param a a-parameter in the Camera Response Function (CRF).
+ * @param b b-parameter in the Camera Response Function (CRF).
+*/
+CV_EXPORTS_W void BIMEF(const Mat& input, Mat& output, float mu=0.5f, float *k=NULL, float a=-0.3293f, float b=1.1258f);
```

Comment:
Did you check that bindings can handle "k" pointer parameter properly?

It would be great to add Python test here: `misc/python/test/test_intensity_transform_bimef.py` (use `self.get_sample()` to fetch test images)

Reply:
I have updated with:
```
CV_EXPORTS_W void BIMEF(InputArray input, OutputArray output, float mu=0.5f, float a=-0.3293f, float b=1.1258f);
CV_EXPORTS_AS(BIMEF2) void BIMEF(InputArray input, OutputArray output, float k, float mu, float a, float b);
```